src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -462,8 +462,14 @@ private function overwrite(string $message): void
         if ($this->overwrite) {
             if (null !== $this->previousMessage) {
                 if ($this->output instanceof ConsoleSectionOutput) {
-                    $lines = floor(Helper::strlen($message) / $this->terminal->getWidth()) + $this->formatLineCount + 1;
-                    $this->output->clear($lines);
+                    $message_lines = explode("\n", $message);
+                    $line_count = \count($message_lines) - $this->formatLineCount + 1;
+                    foreach ($message_lines as $message_line) {
+                        if (Helper::strlen($message_line) / $this->terminal->getWidth() > 1) {
```

Comment:
This is a second bug that's being fixed. If the Terminal width exactly matches the line length, we should not count an extra line. We should only count an extra line if it exceeds the line length.